x/finality/keeper/power_dist_change.go

```diff
@@ -326,6 +350,20 @@ func (k Keeper) ProcessAllPowerDistUpdateEvents(
 		}
 	}
 
+	k.processBtcDelegations(ctx, cacheFpByBtcPkHex, newActiveBtcDels, func(fp, del sdk.AccAddress, sats uint64) {
+		err := k.IncentiveKeeper.BtcDelegationActivated(ctx, fp, del, sats)
+		if err != nil {
+			panic(err)
+		}
+	})
+
+	k.processBtcDelegations(ctx, cacheFpByBtcPkHex, newUnbondedBtcDels, func(fp, del sdk.AccAddress, sats uint64) {
```

Comment:
it seems that `cacheFpByBtcPkHex` is sorted twice in each processor. Can we pass a sorted slice here?

Reply:
`cacheFpByBtcPkHex` is never sorted, it is just a local cache to get FPs by their BTC PK, the slice of BTC delegations that is sorted to make sure it is deterministic